18F/identity-idp
# LG-6193: Enable personal key step in development environment

**Why**: So that a user can functionally complete the proofing process using the new IdV app.

**Testing Instructions:**

1. Navigate to http://localhost:3000
2. Sign in
3. Navigate to http://localhost:3000/verify
4. Complete proofing process up to and including the password entry step
5. After "Continue"-ing from password entry step, observe...
   - Real personal key is shown on page
   - You can enter personal key and continue to post-proofing flow step (account page if session associated with SP, otherwise confirmation screen)

**Implementation Notes:**

- We are excluding users who've opted for GPO verification until LG-6082.
- This is still disabled in all deployed environments, but will now be enabled by default in local development

**Screenshot:**

![localhost_3000_verify_v2_personal_key_confirm_](https://user-images.githubusercontent.com/1779930/164291587-20a759ab-6cc1-4f4c-b40a-abb22f10c73f.png)
